src/environments/environments.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {Environment} from '../environment/environment';
+
+import {ReyServe} from './reyserve';
+
+interface EnvironmentTable {
+  [name: string]: {new(): Environment};
+}
+
+const Environments: EnvironmentTable =([ReyServe]).reduce(
```

Comment:
`reduce` might be a little overkill here. IMO, it's easier to see what's going on with a for..of loop, or simply:

```
environments.set('reyserve', ReyServe);
```